.github/workflows/lint.yml

```
@@ -0,0 +1,22 @@
+name: Lint
+
+on: [pull_request]
```

テストはmainブランチへの差分投入に対しては常に実行してほしいので、以下のようにするとよさそうです。

```suggestion
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```

---

はい！👍